wormhole-connect/src/views/v2/Bridge/Routes/index.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useMemo } from 'react';
+import { useSelector } from 'react-redux';
+import Tooltip from '@mui/material/Tooltip';
+import { makeStyles } from 'tss-react/mui';
+
+import { RoutesConfig } from 'config/routes';
+import useAvailableRoutes from 'hooks/useAvailableRoutes';
+import SingleRoute from 'views/v2/Bridge/Routes/SingleRoute';
+
+import type { Route } from 'config/types';
+import type { RootState } from 'store';
+
+const useStyles = makeStyles()((theme: any) => ({
+  connectWallet: {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    gap: '8px',
+    padding: '8px 16px',
+    borderRadius: '8px',
+    backgroundColor: theme.palette.button.primary,
+    cursor: 'not-allowed',
+    opacity: 0.7,
+    margin: 'auto',
+    maxWidth: '420px',
+    width: '100%',
+  },
+}));
+
+type Props = {
+  selectedRoute: Route | undefined;
+  onRouteChange: (route: Route) => void;
+};
+
+const Routes = (props: Props) => {
+  const { classes } = useStyles();
+
+  const { amount, routeStates } = useSelector(
+    (state: RootState) => state.transferInput,
+  );
+
+  const { sending: sendingWallet, receiving: receivingWallet } = useSelector(
+    (state: RootState) => state.wallet,
+  );
+
+  useAvailableRoutes();
+
+  const supportedRoutes = useMemo(() => {
+    if (!routeStates) {
+      return [];
+    }
+
+    return routeStates.filter((rs) => rs.supported);
+  }, [routeStates]);
+
+  const walletsConnected = useMemo(
+    () => Boolean(sendingWallet.address) && Boolean(receivingWallet.address),
```

Comment:
is this `Boolean()` the same as just doing `!!`?

Reply:
Yes, they are both converting the value to truthy under the hood. It was more of a styling choice, and I can use `!!` for consistency.